addons/sale/static/src/scss/product_configurator.scss

```diff
@@ -47,76 +56,161 @@
 .css_quantity {
     width: initial; // We don't want the quantity form to be full-width
 
-    input[name="add_qty"] {
+    .btn {
+        background: #E8ECEF;
+        color: theme-color('primary');
+        font-weight: normal;
+        padding: 0.7rem;
+        border: none;
+    }
+
+    input {
+        background-color: #F5F9FA;
+        height: auto;
         max-width: 50px;
+        padding: 0.7rem;
         text-align: center;
+        border-color: #E8ECEF;
     }
 }
 
 option.css_not_available {
     color: #ccc;
 }
 
-label.css_not_available {
-    opacity: 0.6;
+select.custom-select.css_attribute_select {
+    background: str-replace(url("data:image/svg+xml;utf8,<svg xmlns='http://www.w3.org/2000/svg' width='175' height='100' fill='#{theme-color('primary')}'><polygon points='0,0 100,0 50,50'/></svg>"), "#", "%23") ;
+    background-size: 25px;
+    background-position: 100% 75%;
+    background-repeat: no-repeat;
+    max-width: 400px;
+}
```

Comment:
@qsm-odoo I changed it in product configurator.. So if we want it for website and product configurator, it should be in Web or Portal no ?
But if I do that Im not certain that it will fit all the cases where a custom select is used (and it will take long time to check)
What I can do is to create a mixin in Portal (or Web ?) so if we want to re-use it later its possible

Reply:
I feel like it should be in web bootstrap_overidden of the frontend part yes... but let's forget about this and see later.